packages/core/contracts/oracle/implementation/test/VotingV2Test.sol

```diff
@@ -39,6 +39,47 @@ contract VotingV2ControllableTiming is VotingV2, Testable {
     function getCurrentTime() public view override(Staker, Testable) returns (uint256) {
         return Testable.getCurrentTime();
     }
+
+    function commitVote(
+        bytes32 identifier,
+        uint256 time,
+        bytes32 hash
+    ) external virtual onlyIfNotMigrated() {
+        commitVote(identifier, time, "", hash);
+    }
+
+    function revealVote(
+        bytes32 identifier,
+        uint256 time,
+        int256 price,
+        int256 salt
+    ) external virtual {
+        revealVote(identifier, time, price, "", salt);
+    }
+
+    function commitAndEmitEncryptedVote(
```

Comment:
moved these methods here. we only need them on the testing side but the UI will always use the interfaces with ancillary data.